Gemfile

```diff
@@ -4,29 +4,28 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.22'
-gem 'rack-cache', '< 1.3.0'
+gem 'rails', '4.1.5'
```

Comment:
Shouldn't this be a newer version?  I thought 4.1.5 was the latest, but it isn't by a mile now - it's vulnerable to a number of security issues.